README.md

```diff
@@ -19,7 +19,14 @@ To build and serve the site, you'll need these tools:
 
 ### Setup
 
-Once you've installed the [prerequisites](#prerequisites), get local packages:
+The[ etcd.io](https://etcd.io) website uses the Docsy Hugo theme. Install the `git` submodule in the root directory of the project with the following command:
```

Comment:
I'd take out the part about installing it into the root of the directory -- The command is being run from the root, but Docsy is getting installed into the `themes` folder.

Instead of linking to etcd.io, it may be better to link to the Docsy website.

nit: spacing around the hyperlink